code/game/verbs/ooc.dm

```diff
@@ -105,7 +105,7 @@ var/global/normal_ooc_colour = "#002eb8"
 	normal_ooc_colour = newColor
 
 /client/verb/round_info()
```

Comment:
This verb could probably be eliminated entirely, and the `map_tag` added to the Status tab, as on TG.